Fabo/export sign message (#4)
luniehq/cosmos-api: README.md

const api = API(STARGATE_URL)

const validators = await api.validators()
```

> ### Create a sign message to sign with on a Ledger or with any other signer

Reviewer: ```suggestion
### Create a message to be signed by a Ledger or with any other signer app
```

Reviewer: does this make sense / is my suggestion true? it was hard to understand precisely what you meant.

```
const { signWithPrivateKey } = require('@lunie/cosmos-keys');
const { createSignMessage } = require('@lunie/cosmos-api');

const stdTx = {
  msg: [
    {
      type: `cosmos-sdk/Send`,
      value: {
        inputs: [
          {
            address: `cosmos1qperwt9wrnkg5k9e5gzfgjppzpqhyav5j24d66`,
            coins: [{ denom: `STAKE`, amount: `1` }]
          }
        ],
        outputs: [
          {
            address: `cosmos1yeckxz7tapz34kjwnjxvmxzurerquhtrmxmuxt`,
            coins: [{ denom: `STAKE`, amount: `1` }]
          }
        ]
      }
    }
  ],
  fee: { amount: [{ denom: ``, amount: `0` }], gas: `21906` },
  signatures: null,
  memo: ``
}

const signMessage = createSignMessage(stdTx, { sequence, accountNumber, chainId });
const signature = signWithPrivateKey(signMessage, Buffer.from(wallet.privateKey, 'hex'));
```

> ### Create a ready to broadcast signed transaction from a message

Author: @AlexBHarley I added this for you. Would this help you?

Reviewer 2: Yes this is great thank you

> ### Create a ready to broadcast signed transaction from a message

Reviewer: ```suggestion
### Create and sign a message that is ready for broadcasting
```

Reviewer: does a message become a transaction when it's signed? 🤔

Author: a transaction is a message wrapped in some meta data ;)

Author: ```suggestion
### Create and sign a transaction from a message which then is ready to be broadcast
```
